recipes/ed25519-blake2b/meta.yaml

```diff
@@ -33,10 +37,8 @@ test:
     - kat-ed25519-blake2b.txt
   commands:
     - pip check
-    - edsig
+    - edsig_blake2b
     - python test_ed25519_kat.py
-  downstreams:
-    - bip-utils
 
```

Comment:
Why is bip-utils no longer downstream?

Reply:
It looks like a bad copy-paste

Reply:
Related Q: Is it a good practice to add the downsstreams? meaning, when I see a dependence is it a good practice to go into the dependent feedstock and add the new downstream?